apps/payment-widgets/src/styles.css

```
@@ -0,0 +1,17 @@
+@tailwind base;
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)